CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ All notable changes to this project are documented in this file.
  - On an `Image`, the default value of `source-clip-width` and `source-clip-height` is now set to
    the size of the image minus the `source-clip-{x,y}`. The source clip size is now used to compute
    the default aspect ratio of the image.
+ - The documented minimum CMake version is now 3.21. The library can still be built with 3.19, but
+   the examples in the documentation require 3.21 on Windows.
```

Comment:
I don't think it is worth adding in the changelog.

Reply:
Ok, done.